src/middlewared/middlewared/plugins/disk.py

```diff
@@ -319,6 +319,9 @@ async def sed_unlock(self, disk_name, disk=None, _advconfig=None):
 
         rv = {'name': disk_name, 'locked': None}
 
+        if osc.IS_LINUX:
```

Comment:
Shouldnt that be after sedutil-cli?

Reply:
@william-gr do you want me to make sedutil debian package now then?

Reply:
No, I thought we already had sedutil and only camcontrol was failing

Reply:
Not sure whats the point in "fixing" this, then? if nothing is wokring anyway?

Reply:
I think Waqar already has a ticket to make SED work

Reply:
There is a report in community slack, pool.format_disks always calls sed_unlock_all and the crash prevents disks from formatting

Reply:
Sorry, ticket about what?

Reply:
Thought we had a ticket about wiping disk not working. If thats not the case, nvm me

Reply:
You have not approved the ticket, is there anything else I am missing?